recipes/dlib-cpp/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "19.17" %}
+
+package:
+  name: dlib-cpp
+  version: {{ version }}
+
+source:
+  url: https://github.com/davisking/dlib/archive/v{{ version }}.tar.gz
+  sha256: 9d2a158b2adad6acba2346f90d929558a691151aa076a0b409ee685534118692
+
+build:
+  skip: true  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - liblapack   # [not win]
+    - libcblas  # [not win]
+    - jpeg
+    - libpng
+  run:
+    - jpeg
+    - libpng
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/dlib/svm.h  # [unix]
+    - test -f ${PREFIX}/lib/libdlib${SHLIB_EXT}  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\dlib\\svm.h exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\dlib.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\dlib.dll exit 1  # [win]
+
+about:
+  home: http://dlib.net/
+  license: Boost 1.0
```

From https://spdx.org/licenses/:
```suggestion
  license: BSL-1.0
```